lib/util/src/condvar.c

```diff
@@ -35,33 +34,35 @@ cv_destroy(struct cv *cv)
 }
 
 int
-cv_timedwait(struct cv *cv, struct mutex *mtx, const int timeout)
+cv_timedwait(struct cv *cv, struct mutex *mtx, const long timeout, const char *wmesg)
 {
-    struct timespec ts;
-    int             rc;
+    int rc;
+
+    hse_wmesg_tls = wmesg ?: "cv_wait";
```

Comment:
Pretty sure this syntax is a GNU extension and we should stop using it.

Reply:
I suspect the world would break if every compiler in existance didn't support this..  But I didn't realize it was a gcc extension...

Reply:
clangd complains about it for me. I need to add clang support to CI. Will get to it today.

Reply:
I'll get rid of both, force wmesg not to be nil.